src/Features/Core/Portable/FindUsages/IFindUsagesContext.cs

```diff
@@ -5,11 +5,12 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Classification;
 using Microsoft.CodeAnalysis.Shared.Utilities;
 
 namespace Microsoft.CodeAnalysis.FindUsages
 {
-    internal interface IFindUsagesContext
+    internal interface IFindUsagesContext : OptionsProvider<ClassificationOptions>
```

Comment:
fwiw, this seems odd to me.  i don't have an intuitive sense of why a find-usages-context (an entity you push found symbols into) would be in any way something you could get classification options from.  

could this not flow in separately somehow?

Reply:
I was wondering the same thing. We already have GetOptionsAsync here. I wanted to scope this PR down a bit to avoid refactoring of all Find Usages. I can follow up on that though - it would totally make sense to separate these concepts.

Reply:
That would very much be my preference.  i have liked how options flow in as a seprate construct in lots of our recent work.

Reply:
Do you want me to do that in this PR or a follow up?

Reply:
can you timebox like an hour to do in this pr?  if it doesn't work out, pulling into another pr is ok (though please do it :D)
.